cg/meta/workflow/analysis.py

```diff
@@ -193,7 +194,7 @@ def add_pending_trailblazer_analysis(self, case_id: str) -> None:
             analysis_type=self.get_application_type(self.status_db.family(case_id).links[0].sample),
             out_dir=self.get_trailblazer_config_path(case_id=case_id).parent.as_posix(),
             config_path=self.get_trailblazer_config_path(case_id=case_id).as_posix(),
-            priority=self.get_priority_for_case(case_id=case_id),
+            priority=self.get_slurm_qos_for_case(case_id=case_id),
```

Comment:
great, in the best of worlds we should also change the variable `priority` to `qos`

Reply:
Done